pkg/exchange/bitget/bitgetapi/v2/get_k_line.go

```diff
@@ -0,0 +1,83 @@
+package bitgetapi
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/c9s/requestgen"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+)
+
+//go:generate -command GetRequest requestgen -method GET -responseType .APIResponse -responseDataField Data
+//go:generate -command PostRequest requestgen -method POST -responseType .APIResponse -responseDataField Data
+
+type KLine struct {
+	// System timestamp, Unix millisecond timestamp, e.g. 1690196141868
+	Ts    types.MillisecondTimestamp
+	Open  fixedpoint.Value
+	High  fixedpoint.Value
+	Low   fixedpoint.Value
+	Close fixedpoint.Value
+	// Trading volume in base currency, e.g. "BTC" in the "BTCUSD" pair.
+	Volume fixedpoint.Value
+	// Trading volume in quote currency, e.g. "USD" in the "BTCUSD" pair.
+	QuoteVolume fixedpoint.Value
```

Comment:
I guess this could overflow? if we multiply BTC price with the day volume?

Reply:
No, I obtained the trade volume of Binance last 24h, which was $1,628,168,452, and when multiplied by $50,000, it is less than the int64 range (9,223,372,036,854,775,807).